### PR TITLE
Enable pulse build in travis

### PR DESCRIPTION
**Pull request by reactormonk**

### CODE REVIEW

**Comment by reactormonk** [comment]
Anyone got time for a review?

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,11 @@
 language: c
+addons:
+  apt:
+    packages:
+      libpulse-dev
 script:
   - ./autogen.sh
-  - ./configure
+  - ./configure --prefix=/usr
```
Comment (rillian) [inline .travis.yml]: Can you say why this change is necessary? Please also include your motivation in the commit message.

Comment (reactormonk) [reply .travis.yml]: Apparently it isn't.

Comment (rillian) [reply .travis.yml]: Great. Thanks for fixing.

**Comment by rillian** [comment]
Merged in df53ae03eb72ea6e6a1d5d2cda911b11e4c13bfd.

**Comment by rillian** [comment]
Thanks for the patch! :)
